modules/tools/record_play/rtk_player.py

```diff
@@ -223,6 +223,13 @@ def publish_planningmsg(self):
             adc_point.path_point.theta = self.data['theta'][i]
             adc_point.path_point.s = self.data['s'][i]
 
+            if CHANGE_TO_COM:
```

Comment:
load values from config to be adaptive to different vehicles. https://github.com/ApolloAuto/apollo/blob/master/modules/common/data/vehicle_param.pb.txt